goci-interfaces/goci-ui/src/main/resources/static/js/common-datatables/associations-datatable.js

```diff
@@ -345,3 +356,22 @@ function pValueSorter(a, b) {
     if (parseInt(value1[0]) < parseInt(value2[0])) return -1;
     return 0;
 }
+
+/*
+ This generates the file name using the date, table type and page identifier
+  */
+function getFilename(table) {
+    // get current date
+    var d = new Date();
+    var curr_date = d.getDate();
+    var curr_month = d.getMonth() + 1;
+    curr_month = curr_month < 10 ? '0' + curr_month : curr_month;
+
+    var curr_year = d.getFullYear();
+    var date = `${curr_year}-${curr_month}-${curr_date}`;
+    // Set other parameters:
+    var pathArray = window.location.pathname.split('/');
+    var docType = pathArray[2];
+    var query = pathArray[3];
```

Comment:
I am not sure what does that mean.

Reply:
Well, rather than say parse "EFO_0002690" from the page URL it(the page entity,e.g. EFO_0002690 or rs7329174, etc.) potentially could be passed to the function, but after looking back this seems the easiest way to get that information so should be ok. I know Cinzia mentioned a strong preference to not use "window.location" in the code Xin wrote so just passing this along.